packages/server/graphql/mutations/addPokerTemplate.ts

```diff
@@ -53,6 +54,11 @@ const addPokerTemplate = {
     if (!viewerTeam) {
       return standardError(new Error('Team not found'), {userId: viewerId})
     }
+    const viewer = await getUserById(viewerId)
```

Comment:
-1 Let's use the dataloader here `await dataloader.get('users').loadNonNull(viewerId)` just in case we load the viewer for other reasons in the same resolve call.